packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -419,6 +670,7 @@ class HtmlElementView extends StatelessWidget {
   ///
   /// [onElementCreated] is called when the DOM element is created. It can be
   /// used by the app to customize the element by adding attributes and styles.
+  /// This method is called *before* the element is injected into the DOM.
```

Comment:
Not sure if this was meant to be a standalone paragraph like in the method below, but if it was, then there needs to be one empty line between paragraphs.

Reply:
Nope, just a continuation, it just landed near the ~80 char line and broke it down to a new line.